custom-connectors/CopilotForFinanceCommunications/apiDefinition.swagger.json

```diff
@@ -1,765 +1,1376 @@
+            "name": "contactId",
+            "required": true,
+            "type": "string"
           },
           {
-            "$ref": "#/parameters/apiVersion"
+            "in": "query",
+            "name": "language",
+            "description": "The language code is in BCP 47 format.",
+            "type": "string"
           },
           {
-            "name": "body",
             "in": "body",
+            "name": "body",
             "required": true,
             "schema": {
-              "type": "object",
-              "properties": {
-                "content": {
-                  "type": "string",
-                  "title": "Content",
-                  "description": "Message content."
-                },
-                "senderDisplayName": {
-                  "type": "string",
-                  "title": "Name",
-                  "description": "Display name for the message sender."
-                }
-              },
-              "required": [
-                "content",
-                "senderDisplayName"
-              ]
+              "$ref": "#/definitions/ContactForUpdate"
             }
           }
-        ]
-      }
-    },
-    "/chat/threads/{chatThreadId}/participants/:add": {
-      "post": {
+        ],
         "responses": {
           "201": {
-            "description": "Success",
+            "description": "The request has succeeded and a new resource has been created as a result."
+          },
+          "default": {
+            "description": "An unexpected error response.",
             "schema": {
-              "type": "object",
-              "properties": {
-                "invalidParticipants": {
-                  "type": "array",
-                  "items": {
-                    "type": "object",
-                    "properties": {
-                      "target": {
-                        "type": "string",
-                        "description": "Target ID.",
-                        "x-ms-summary": "Target"
-                      },
-                      "code": {
-                        "type": "string",
-                        "description": "Error code.",
-                        "x-ms-summary": "Code"
-                      },
-                      "message": {
-                        "type": "string",
-                        "description": "Error message.",
-                        "x-ms-summary": "Message"
-                      }
-                    }
-                  },
-                  "description": "Invalid participants list."
-                }
-              }
+              "$ref": "#/definitions/Error"
             }
-          },
-          "401": {
-            "description": "Unauthorized"
-          },
-          "403": {
-            "description": "Forbidden"
           }
-        },
-        "summary": "Add chat thread participants",
-        "description": "Add participants to chat thread.",
-        "operationId": "AddParticipants",
```

Comment:
The operation has been deleted. This change breaks existing flows. ValuePath: operation/AddParticipants

Reply:
We are adding a new template for a custom connector, so nothing is actually deleted.
Endpoint is deleted compared to prior iteration because prior iteration had incorrect swagger included.